hack/local-up-cluster.sh

```diff
@@ -93,7 +93,7 @@ sudo "${GO_OUT}/kube-apiserver" \
   --address="${API_HOST}" \
   --port="${API_PORT}" \
   --runtime_config=api/v1beta3 \
-  --etcd_servers="http://localhost:4001" \
+  --etcd_servers="http://127.0.0.1:4001" \
```

Comment:
This was changed when upgrading to 2, and I'm reverting it back now that we're using -bind-addr